From aa35362aa2835e9e2903f5f785f10402a6abd3a1 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Sun, 17 Feb 2019 16:58:48 +0100
Subject: [PATCH] Fix solid-lite static linking

If BUILD_SHARED_LIBS is part of standard system cmake args:

  cantata: error while loading shared libraries: libsolidlite.so:
  cannot open shared object file: No such file or directory
---
 3rdparty/solid-lite/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/3rdparty/solid-lite/CMakeLists.txt b/3rdparty/solid-lite/CMakeLists.txt
index 4ae51aab..86e37189 100644
--- a/3rdparty/solid-lite/CMakeLists.txt
+++ b/3rdparty/solid-lite/CMakeLists.txt
@@ -202,7 +202,7 @@ if(APPLE)
    set(solidlite_OPTIONAL_LIBS ${IOKIT_LIBRARY})
 endif()
 
-add_library(solidlite ${solidlite_LIB_SRCS})
+add_library(solidlite STATIC ${solidlite_LIB_SRCS})
 
 if ( UDEV_FOUND )
    set(solidlite_OPTIONAL_LIBS ${solidlite_OPTIONAL_LIBS} ${UDEV_LIBS})
-- 
2.20.1

